satellite/contact/service.go

```diff
@@ -23,14 +23,16 @@ var mon = monkit.Package()
 type Service struct {
 	log       *zap.Logger
 	overlay   *overlay.Service
+	peerids   overlay.PeerIdentities
 	transport transport.Client
 }
 
 // NewService creates a new contact service
```

Comment:
maybe info from the PR description could be added here so it's more apparent to a code-skimmer what this service is for?